packages/@eventual/aws-runtime/src/env.ts

```diff
@@ -11,6 +11,7 @@ export const ENV_NAMES = {
   EVENT_BUS_ARN: "EVENTUAL_EVENT_BUS_ARN",
   WORKFLOW_QUEUE_URL: "EVENTUAL_WORKFLOW_QUEUE_URL",
   TASK_TABLE_NAME: "EVENTUAL_TASK_TABLE_NAME",
+  OPENSEARCH_ENDPOINT: "OPENSEARCH_ENDPOINT",
```

Comment:
The rest of them have `EVENTUAL_` to avoid overlap with a user's env variables.

Reply:
Yeah I was gonna fix that.